docs: add README for Airflow DAG sample, update code formatting in other docs (#722)
GoogleCloudPlatform/professional-services-data-validator: samples/airflow/README.md

> ### Warning 

Reviewer: Let's make this README similar to the [Cloud Run](https://github.com/GoogleCloudPlatform/professional-services-data-validator/tree/develop/samples/run) or [Cloud Functions](https://github.com/GoogleCloudPlatform/professional-services-data-validator/tree/develop/samples/functions) samples. We should add a title 'Data Validation on Airflow' and explain how to deploy this on Airflow/Composer. 

Under the high level overview we can include this 'Warning' section. That way the README is a more complete documentation for the Airflow sample code.

### Warning 

The Airflow DAG code expects the raw config JSON only, 

> it does not support a JSON format that was generated by converting an existing

Reviewer: We can make lines 4-6 more concise:

```suggestion
The Airflow DAG expects the raw config JSON which is not the same as a YAML config converted to JSON.
```

YAML config file.

Parameters from YAML format are slightly different from the JSON config,

> there is an example for Cloud Run [here](https://github.com/GoogleCloudPlatform/professional-services-data-validator/tree/develop/samples/run#test-cloud-run-endpoint) 

Reviewer: Let's add some more detail like so:

Parameters in a typical YAML config file are slightly different from the raw JSON config, which is generated after DVT parses the YAML. The [build_config_manager() method ](https://github.com/GoogleCloudPlatform/professional-services-data-validator/blob/develop/data_validation/config_manager.py#L425) generates the JSON config and should be used as a reference.

Our Cloud Run sample also expects a raw JSON config in the `'data'` variable shown [here](https://github.com/GoogleCloudPlatform/professional-services-data-validator/tree/develop/samples/run#test-cloud-run-endpoint)

that shows the expected JSON config in the `'data'` variable.

> For example, check the following YAML config part.

Reviewer: ```suggestion
For example, the following YAML config is equivalent to the JSON config below. 
```

```yaml
result_handler: {}
source: my_bq_conn
target: my_bq_conn
validations:
- aggregates:
  - field_alias: count
    source_column: null
    target_column: null
    type: count
...
```

This YAML is equivalent to the JSON part below, where the source param is written as `source_conn`.

```json
{

>     "source_conn": BQ_CONN,

Reviewer: Some of this text on this line and the ones below are highlighted.. is there a way to fix that?

# Data Validation on Airflow (ideally Cloud Composer)

DVT on Airflow makes use of our [latest released version of DVT library on PyPi](https://pypi.org/project/google-pso-data-validator/#history).
There is no specific operator for it at the moment, so the DAG uses the
main structures of the library itself.

The input is a JSON data representation of the DVT run configuration. 

By default, the DAG will output the results to BigQuery as a result handler.

### Requirements to run the DAG:
- Pre-existing Airflow environment created with Public IP (Private environment disabled)

> - Create an Airflow variable called `gcp_project with the GCP Project ID

Reviewer: ```suggestion
- Create an Airflow variable called `gcp_project with the GCP Project ID`
```

### Instructions

> 1. Download the Python file with the DAG

Reviewer: ```suggestion
1. Download the DAG file in this directory
```

> 2. Update the JSON configuration to your use case (source, target tables, etc)

Reviewer: ```suggestion
2. Update the JSON configuration for your use case (update table names, etc.)
```

> 3. Upload it to the DAGs folder of your Airflow environment

Reviewer: ```suggestion
3. Upload it to the DAGs folder in your Airflow environment
```
